packages/vats/src/core/utils.js

```diff
@@ -12,8 +12,8 @@ const mapEntries = (obj, f) =>
   // @ts-ignore entries() loses key type
   fromEntries(entries(obj).map(([p, v]) => f(p, v)));
 
-// TODO: phase out ./issuers.js
 export const CENTRAL_ISSUER_NAME = 'RUN';
+export const ORACLE_PRICE_BRAND_NAME = 'USD';
```

Comment:
It could be used in brand or elsewhere. 
```suggestion
export const ORACLE_PRICE_NAME = 'USD';
```

Reply:
No, it can only be used as a brand.  I'll rename to `ORACLE_BRAND_PRICE_NAME`.